Example/ParalayoutSnapshotTests/ViewDistributionSnapshotTests.swift

```diff
@@ -45,6 +45,30 @@ final class ViewDistributionSnapshotTests: SnapshotTestCase {
         assertSnapshot(matching: containerView, as: .image, named: nameForSnapshot(with: ["vertical"]))
     }
 
+    func testDistributionBuilder() {
```

Comment:
these test are carbon copies of the test directly above, with the following changes:
1. Added the word `Builder` after `Distribution` in the test title.
2. Used the result builder distribution api rather than the array API